app/src/main/res/drawable/item_in_history_indicator_background.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+    <gradient
+        android:angle="-90"
+        android:endColor="#B2000000"
+        android:startColor="#E6000000"/>
```

Comment:
I'd prefer the shadow to be more transparent, something like this:
```suggestion
        android:endColor="#6C000000"
        android:startColor="#A0000000"/>
```

Also, what about making the color different based on the theme? Imo the black shadow looks a little bit strange in the white theme, but maybe it's just an impression.

Reply:
> I'd prefer the shadow to be more transparent

IMO, that much more transparency makes it hard to read when the thumbnail doesn't use darker colors.

But I do like to see the thumbnail a little better...

What about making the gradient a little lighter towards the end, what do you say on a 90% → 40% gradient instead of the current 90% → 70%?

> Also, what about making the color different based on the theme? 

B&W or some other color?

> Imo the black shadow looks a little bit strange in the white theme, but maybe it's just an impression.

To me it isn't much different/stranger than any other dark thumbnail.

Reply:
> B&W or some other color?

I was thinking about B&W, but then you are right, there is actually no need for this.
So, let's use a `90% → 40%` black shadow, it seems the best option.